features/storage/kvstore/conf/tdb_internal/mbed_lib.json

```diff
@@ -9,5 +9,11 @@
             "help": "If default, the base address is set to the first sector after the application code ends.",
             "value": "0"
         }
+    },
+    "target_overrides": {
+        "FUTURE_SEQUANA_M0_PSA": {
+            "internal_size": "0x8000",
+            "internal_base_address": "0x10078000"
```

Comment:
This seems to be a wrong base address.
M0+ flash region is 0x10000000 - 0x10078000.
I think we need base address of 0x10070000.

Reply:
After discussing this within the psa core team, we had a fix in is_buffer_accessible() which resolved the memory issue.